ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/push-queries.json

```diff
@@ -927,6 +927,66 @@
           {"finalMessage":"Limit Reached"}
         ]}
       ]
+    },
+    {
+      "name": "NULL Arithmetic Behavior - INTEGER addition",
```

Comment:
instead of RQTT I think it makes sense to have this in our normal QTT tests `math.json` would be a good fit. They're much lighter weight and also have historical tests

Reply:
the tests in https://github.com/confluentinc/ksql/blob/7523e1e5c2d850a1f0d210d1680e6307ef351697/ksqldb-engine/src/test/java/io/confluent/ksql/execution/ExpressionEvaluatorParityTest.java ensure that